apps/address-book/contracts/AddressBook.sol

```diff
@@ -72,13 +75,29 @@ contract AddressBook is AragonApp {
         emit EntryRemoved(_addr);
     }
 
+    /**
+     * @notice Remove address `_addr` from the registry.
```

Comment:
is this radspec correct?

Reply:
Can we return the old/new CID in the radspec? if this goes to the voting app as is, there is no context for the change